src/pods/edit-realtion/edit-relation.business.ts

```diff
@@ -7,14 +7,14 @@ import {
   RelationVm,
   TableVm,
 } from '@/core/providers/canvas-schema';
-import { RelationFormVm } from './edit-relation.vm';
+import { FormValueVm, RelationFormVm } from './edit-relation.vm';
 
-// TODO: Test - this mapper needs to be updated when an ID relation appears.
 export const mapRelationFormVmToRelaionVM = (
-  values: RelationFormVm
+  values: RelationFormVm,
```

Comment:
Add a TODO and reference this case for unit test implementation

https://github.com/Lemoncode/mongo-modeler/issues/145

Reply:
We already have a comment for testing, the issue is #118.